covid_graphs/covid_graphs/scripts/country_dashboard.py

```diff
@@ -27,11 +71,7 @@ def create_graphs(
     return country_graphs
 
 
-def create_dashboard(
-    data_dir: Path, server: Flask,
-):
-    # TODO(miskosz): Don't use print.
-    print("Creating dashboard for prediction graphs.")
+def _prepare_data_structures(data_dir: Path):
```

Comment:
Here I would add a description of the return value, at least as a type annotation:
```
-> Tuple[Dict[str, PredictionEvent], Dict[str, List[CountryGraph]]]
```

Pro tip: You can define a type alias somewhere in the global scope at the top of the file (below imports), e.g.:
```
CountryGraphsByReportName = Dict[str, List[CountryGraph]]
```

Reply:
Done